lib/schema.js

```diff
@@ -43,6 +50,20 @@ module.exports = [
       }
     }
   },
+  {
+    name: '_linkFile',
+    label: 'Link File',
+    type: 'joinByOne',
+    withType: 'apostrophe-file',
+    idField: 'fileId',
+    required: true,
+    projection: {
+      title: 1,
+      slug: 1,
```

Comment:
Use _url: 1 to replace all the other fields you listed here, except title

Reply:
👍